addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -166,12 +169,25 @@ public void handleCommand(final ChannelUID channelUID, final Command command) {
             case ChromecastBindingConstants.CHANNEL_VOLUME:
                 handleVolume(command);
                 break;
+            case ChromecastBindingConstants.CHANNEL_PLAY_URI:
+                handlePlayUri(command);
+                break;
             default:
                 logger.debug("Received command {} for unknown channel: {}", command, channelUID);
                 break;
         }
     }
 
+    private void handlePlayUri(Command command) {
+        if (command instanceof StringType) {
+            if (command.equals("NULL")) {
```

Comment:
> Do we really need this functionality? 

Not necessarily.

> why not just send a PAUSE to the controls channels?

Yes, that was also my initial thought as well. But then I thought that a PAUSE still keeps the current url set up in the Chromecast and maybe you would want to clear it - that's why I thought sending an empty string should be accepted as well.

> though the inbuilt REST client doesn't allow you to send an empty body to a POST.

Ok, that's a fair point - having an empty string as a serialisation is indeed a bit weird; so after all, I'd be fine with simply removing this here completely and only offer the PAUSE command on the control :-)